src/date-picker/date-picker.scss

```diff
@@ -2,10 +2,10 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 @import '../icon/index';
+@import '../menu/index';
 
 @mixin iui-date-picker {
-  @include iui-reset;
-  @include iui-font-family;
+  @include iui-dropdown;
```

Comment:
This is using the same mixin used on the dropdown menu & column filter menu.

Reply:
What does it give in this case?

Reply:
Not a whole lot, but it gives [the properties we want to be consistent across all dropdowns](https://github.com/iTwin/iTwinUI/blob/main/src/menu/menu.scss#L7):
- Reset
- Font family
- Background color
- Drop shadow